deliveries/TREX_Network_milestone_1.md

```diff
@@ -0,0 +1,31 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** 
+[TREX Network](https://github.com/w3f/Grants-Program/blob/master/applications/TREX_Network.md)
```

Comment:
Hi @yangfh2004, thanks for the delivery, someone of the team will take care of it soon.

In the meantime, could you please make this little adjustment in the delivery document?
```suggestion
* **Application Document:** [TREX Network](https://github.com/w3f/Grants-Program/blob/master/applications/TREX_Network.md)
```